README.md

```diff
@@ -214,6 +214,8 @@ requirements. See table below.
 | Plural module | cldr/supplemental/plurals.json (for cardinals)<br>cldr/supplemental/ordinals.json (for ordinals) |
 | Relative time module | cldr/main/`locale`/dateFields.json<br>+CLDR JSON files from number and plural modules |
 
+As alternative to deducing this yourself you could try using a tool called [Globalize · So What'cha Want](http://johnnyreilly.github.io/globalize-so-what-cha-want/).  This allows you to select the modules you're interested in using and tell you the Globalize files *and* CLDR JSON that you need. 
```

Comment:
Thanks @kswedberg.

I'm also wondering if we can simplify the sentence... Something like this:

> As an alternative to deducing the CLDR requirements yourself, use this [online tool][]. The tool allows you ... and tells you ...